modules/manually-gathering-logs-without-ssh.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *installing/installing-gather-logs.adoc
+
+[id="installation-manually-gathering-logs-without-SSH_{context}"]
+= Manually gathering logs without SSH access to your host(s)
+
+Manually gather logs in situations where `must-gather` or automated collection
+methods do not work.
+
+If you do not have SSH access to your node, you can access the systems journal
+to investigate what is happening on your host.
+
+.Prerequisites
+
+* Your {product-title} installation must be complete.
+* Your API service is still functional.
+* You have system administrator privileges.
+
+.Procedure
+
+. Access `journald` unit logs under `/var/log` by running:
++
+----
+$ oc adm node-logs --role=master -u kubelet
+----
+
+. Access host file paths under `/var/log` by running:
++
+----
+$ oc adm node-logs --role=master --path=openshift-apiserver
```

Comment:
Is this something we expect?
```
[root@gpei-new-slave auth]#17138 oc adm node-logs --role=master --path=openshift-apiserver
gpei-5-drkmm-m-0.c.openshift-qe.internal audit.log
gpei-5-drkmm-m-1.c.openshift-qe.internal audit.log
gpei-5-drkmm-m-2.c.openshift-qe.internal audit.log
```
When I'm trying with `oc adm node-logs --role=master --path=openshift-apiserver/audit.log`, it will print out the detailed log content.

Reply:
@sferich888 can you please confirm?

Reply:
Yes, it gets all the logs from that directory.